app/controllers.rb

```diff
@@ -9,13 +9,17 @@
   end
 end
 
+CACHE_TTL = 300
+
 # All API requests are prefixed by the API version
 # in this case, "v1" - e.g. "/vi/endpoints" etc.
 OpenDataMaker::App.controllers :v1 do
   before do
     content_type :json
     headers 'Access-Control-Allow-Origin' => '*',
-            'Access-Control-Allow-Methods' => ['GET']
+            'Access-Control-Allow-Methods' => ['GET'],
+            'Surrogate-Control' => "max-age=#{CACHE_TTL}"
```

Comment:
It's probably fine, but it didn't seem like you needed both Surrogate-Control and Cache-Control unless you wanted them to be different.  The latter is sufficient for everyone.